cszczepaniak/cribbly-frontend
# Dev tools and page layout

# What I added (link any issues addressed)
- installed prettier, eslint, lint-staged, and husky to format and lint our code whenever we commit
  - you can bypass this by passing `-n` when you commit but avoid this so our code formatting stays consistent
- set up an initial page layout (app bar and page container). it can completely change if we want it to, but I wanted to get something going 

# How to test it
- first, `npm install`
- for the dev tools, you can try creating lint warnings (create an unused variable is an easy one) and try committing. you should get yelled at and your change shouldn't be committed
- for the page layout, start the app and look at the page. click on the "my team" link and see how that looks - the app bar and the padding/spacing/everything else is in PageWrapper, so it should stay consistent across pages
